doc/adr/0001-prevent-tables-from-being-creating-simultaneously-in-cassandra-via-a-locks-table.md

```diff
@@ -0,0 +1,31 @@
+# 1. Prevent tables from being creating simultaneously in cassandra via a locks table
+
+Date: 18/05/2016
```

Comment:
At the risk of starting a flame war: since we're distributed across two continents, would it be clearer to standardize on, say, ISO 8601 for dates?